internal/generator/vector/elements/elements.go

```diff
@@ -37,10 +37,23 @@ type Remap struct {
 	VRL         string
 }
 
+type RemapCW struct {
```

Comment:
Do not add anything feature/output specific in this package. this package is the base package.

Reply:
please remove this struct